File: KMTracker/KMTracker/Rit.cs
using System.Collections.Generic;

namespace KMTracker
{
	public class Rit
	{

		public int Id
		{
			get;
			set;
		}

		public string Description
		{
			get;
			set;
		}

		public List<Coordinate> Coordinates
		{
			get;
			set;
		}

		public Rit(string description, List<Coordinate> coordinates)
		{
			Description = description;
			Coordinates = coordinates;
		}
	}
}

File: Server/api/models/Rit.js
/**
 * Rit.js
 *
 * @description :: TODO: You might write a short summary of how this model works and what it represents here.
 * @docs        :: http://sailsjs.org/documentation/concepts/models-and-orm/models
 */

module.exports = {

  autoUpdatedAt: false,
  createdAt: false,

  attributes: {

    id: {
      type: 'integer',
      primaryKey: true,
    },

    car:{
      model: 'car'
    },

  	description:{
  		type: 'string'
  	},

    // Add reference to
    coordinates: {
  		collection: 'coordinate',
      via: 'rit'
  	}

  }
};

